test/interaction.js

```diff
@@ -413,6 +421,103 @@
 			});
 		});
 
+		describe('simulate tabbing using native focus()', function() {
+
+			describe('defaults', function() {
+				var test, input1, input2;
+
+				before(function(done) {
+					test = setup_test('<select>' +
+						'<option value="">No selection</option>' +
+						'<option value="a">A</option>' +
+						'<option value="b">B</option>' +
+						'</select>', {});
+					input1 = $('<input type="text" class="first">');
+					input2 = $('<input type="text" class="last">');
+					test.$select.parent().prepend(input1);
+					test.$select.parent().append(input2);
+					done();
+				});
+
+				after(function() {
+					input1.remove();
+					input2.remove();
+				});
+
+				it('should give the control focus', async function() {
+					await tabTo(input1[0]);
+					expect(test.selectize.isFocused).to.be.equal(false);
+					await tabTo(test.selectize.$control_input[0]);
+					expect(test.selectize.isFocused).to.be.equal(true);
+				});
+
+				it('should remove the control focus', async function() {
+					await tabTo(test.selectize.$control_input[0]);
+					expect(test.selectize.isFocused).to.be.equal(true);
+					await tabTo(input2[0]);
+					expect(test.selectize.isFocused).to.be.equal(false);
+				});
+
+				it('should open the control', async function() {
+					await tabTo(input1[0]);
+					expect(test.selectize.isOpen).to.be.equal(false);
+					await tabTo(test.selectize.$control_input[0]);
+					expect(test.selectize.isOpen).to.be.equal(true);
+				});
+
+				it('should close the control', async function() {
+					await tabTo(test.selectize.$control_input[0]);
+					expect(test.selectize.isOpen).to.be.equal(true);
+					await tabTo(input2[0]);
+					expect(test.selectize.isOpen).to.be.equal(false);
+				});
+
+				// TODO: this would work if tabTo was using actual keyboard interactions,
+				// and not just focus()
+				xit('should select the first value on blur', async function() {
+					await tabTo(test.selectize.$control_input[0]);
+					await tabTo(input2[0]);
+					expect(test.selectize.getValue()).to.be.equal('a');
+				});
```

Comment:
This is the test that would require a better emulation/control over the headless browser (eg. puppeteer).